oap-server/server-cluster-plugin/cluster-kubernetes-plugin/src/main/java/org/apache/skywalking/oap/server/cluster/plugin/kubernetes/NamespacedPodListInformer.java

```diff
@@ -83,7 +83,7 @@ private void doStartPodInformer(ClusterModuleKubernetesConfig podConfig) throws
         SharedIndexInformer<V1Pod> podSharedIndexInformer = factory.sharedIndexInformerFor(
             params -> coreV1Api.listNamespacedPodCall(
                 podConfig.getNamespace(), null, null, null, null,
-                podConfig.getLabelSelector(), Integer.MAX_VALUE, params.resourceVersion, params.timeoutSeconds,
+                podConfig.getLabelSelector(), Integer.MAX_VALUE, params.resourceVersion, 300,
```

Comment:
300 seconds? I am a little confused.

Reply:
![image](https://user-images.githubusercontent.com/31562192/97956001-8b312400-1de2-11eb-98b8-44d0ee09fb66.png)
Follow the Kubernetes default configuration

Reply:
https://github.com/kubernetes-client/java/pull/1344/files

The default value is indeed 300s

Reply:
OK, got it now. Could you submit an issue to track this? 

I think we need to upgrade the client to 10.1(maybe? the Last release is 10.0 already), and change this back to the right way.

Reply:
https://github.com/apache/skywalking/issues/5778